charts/jaeger/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Can we put this chart inside the top level `jaeger` directory?  e.g. `jaeger/charts/jaeger/Chart.yaml`?